### PR TITLE
Add error messages on software installations

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

To be marged after 9.3 release since it adds new strings.

### CODE REVIEW

File: front/computer_softwareversion.form.php [hunk 1]
```diff
@@ -49,6 +49,15 @@
                     //TRANS: %s is the user login
                     sprintf(__('%s installs software'), $_SESSION["glpiname"]));
       }
+   } else {
+       $message = null;
+       if (!isset($_POST['softwares_id']) || !$_POST['softwares_id']) {
```
Comment (cedric-anne) [inline front/computer_softwareversion.form.php]: Maybe I missed something, but I think `softwares_id` is not used here. Did you mean `computers_id` ?

Comment (trasher) [reply front/computer_softwareversion.form.php]: You have to select a software (`softwares_id`) and then a version (`softwareversions_id`). This is indeed not used directly, but it's not possible to select a version if you did not select a software.

Comment (trasher) [reply front/computer_softwareversion.form.php]: Page is called from a computer; si this is not possible to have an empty id. User have to select a soft and its version, and can click if it's empty.

Comment (cedric-anne) [reply front/computer_softwareversion.form.php]: ok

**Pull request merged**